plugins/slicer/MONAILabel/MONAILabel.py

```
@@ -1303,19 +1303,57 @@ def onNextSampleButton(self):
+                            raise ValueError(f"multi_file=True but path is not a directory: {dir_path}")
+
+                        # get valid image paths
+                        entries = sorted(os.listdir(dir_path))
+                        image_paths = []
+                        for name in entries:
+                            full_path = os.path.join(dir_path, name)
+                            if os.path.isfile(full_path):
+                                image_paths.append(full_path)
+
+                        nodes = []
+                        for idx, image in enumerate(image_paths):
```

Ruff (0.15.2)

[warning] Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)